### PR TITLE
[Docs] Update README

### PR DESCRIPTION
**Pull request by Xiangxu-0103**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

- Update tutorial link
- Update Chinese version of README

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: README_zh-CN.md [hunk 1]
```diff
@@ -38,35 +38,35 @@
 
 ## 简介
 
-MMEngine 是一个基于 PyTorch 用于深度学习模型训练的基础库，支持在 Linux、Windows、macOS 上运行。它具有如下三个亮点：
+MMEngine 是一个用于训练基于 PyTorch 深度学习模型的基础库。它提供了坚实的工程基础，使开发人员无需在工作流上编写冗余代码。作为 OpenMMLab 所有代码库的训练引擎，其在不同研究领域支持了上百个算法。此外，MMEngine 也可以用于非 OpenMMLab 项目中。
```
Comment (HAOCHENYE) [inline README_zh-CN.md:41]:
```suggestion
MMEngine 是一个基于 PyTorch 实现的，用于训练深度学习模型的基础库。它为开发人员提供了坚实的工程基础，以此避免在在工作流上编写冗余代码。作为 OpenMMLab 所有代码库的训练引擎，其在不同研究领域支持了上百个算法。此外，MMEngine 也可以用于非 OpenMMLab 项目中。
```

**Review by HAOCHENYE — DISMISSED** [review]

File: README_zh-CN.md [hunk 2]
```diff
@@ -18,9 +18,9 @@
   </div>
   <div>&nbsp;</div>
 
-[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/mmengine)](https://pypi.org/project/mmengine/)
-[![PyPI](https://img.shields.io/pypi/v/mmengine)](https://pypi.org/project/mmengine)
-[![license](https://img.shields.io/github/license/open-mmlab/mmengine.svg)](https://github.com/open-mmlab/mmengine/blob/main/LICENSE)
+[![PyPI - Python Version](http://img.shields.io/pypi/pyversions/mmengine)](https://pypi.org/project/mmengine/)
+[![PyPI](http://img.shields.io/pypi/v/mmengine)](https://pypi.org/project/mmengine)
+[![license](http://img.shields.io/github/license/open-mmlab/mmengine.svg)](https://github.com/open-mmlab/mmengine/blob/main/LICENSE)
```
Comment (zhouzaida) [inline README_zh-CN.md:23]: Why are these `https` replaced with `http`?

**Comment by Xiangxu-0103** [comment]
> Hi, please update the code with the latest `main` and resolve the conflicts.

resolved

**Review by HAOCHENYE — APPROVED** [review]
Approved!!!
image

**Comment by zhouzaida** [comment]
Hi @Xiangxu-0103 , could you help update the introduction in https://github.com/open-mmlab/mmengine/blob/main/docs/zh_cn/get_started/introduction.md#%E4%BB%8B%E7%BB%8D?

**Comment by Xiangxu-0103** [comment]
> Hi @Xiangxu-0103 , could you help update the introduction in https://github.com/open-mmlab/mmengine/blob/main/docs/zh_cn/get_started/introduction.md#%E4%BB%8B%E7%BB%8D?

updated

File: README_zh-CN.md [hunk 3]
```diff
@@ -38,25 +38,25 @@
...
+1. **通用且强大的执行器**：
 
-2. 统一：MMEngine 设计了一个接口统一的开放架构，使得:
+   - 支持用少量代码训练不同的任务，例如仅使用 80 行代码就可以训练 ImageNet（原始 PyTorch 示例需要 400 行）。
+   - 轻松兼容流行的算法库（如 TIMM、TorchVision 和 Detectron2）中的模型。
 
-   - 用户可以仅依赖一份代码实现所有任务的轻量化，例如 MMRazor 1.x 相比 MMRazor 0.x 优化了 40% 的代码量
+2. **接口统一的开放架构**：
+
+   - 使用统一的 API 处理不同的算法任务，例如，实现一个方法并应用于所有的兼容性模型。
```
Comment (zhouzaida) [inline README_zh-CN.md:52]:
```suggestion
   - 使用统一的接口处理不同的算法任务，例如，实现一个方法并应用于所有的兼容性模型。
```

File: docs/zh_cn/get_started/introduction.md [hunk 4]
```diff
@@ -1,24 +1,22 @@
...
-   - 支持用少量代码训练不同的任务，例如仅使用 80 行代码就可以训练 imagenet（pytorch example 400 行）
-   - 轻松兼容流行的算法库如 TIMM、TorchVision 和 Detectron2 中的模型
+   - 支持用少量代码训练不同的任务，例如仅使用 80 行代码就可以训练 ImageNet（原始 PyTorch 示例需要 400 行）。
+   - 轻松兼容流行的算法库（如 TIMM、TorchVision 和 Detectron2）中的模型。
 
-2. 统一：MMEngine 设计了一个接口统一的开放架构，使得
+2. **接口统一的开放架构**：
 
-   - 用户可以仅依赖一份代码实现所有任务的轻量化，例如 MMRazor 1.x 相比 MMRazor 0.x 优化了 40% 的代码量
+   - 使用统一的 API 处理不同的算法任务，例如，实现一个方法并应用于所有的兼容性模型。
```
Comment (zhouzaida) [inline docs/zh_cn/get_started/introduction.md:12]:
```suggestion
   - 使用统一的接口处理不同的算法任务，例如，实现一个方法并应用于所有的兼容性模型。
```

**Review by zhouzaida — APPROVED** [review]

**Pull request merged**